default.xml

```diff
@@ -21,7 +21,7 @@
     <copyfile src="core/root.mk" dest="Makefile" />
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit